winui/SfShimmer/Overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Overview of WinUI SfShimmer control | Syncfusion
+description: Learn about the introduction of Syncfusion WinUI Shimmer Control (SfShimmer) with essential features and more.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# WinUI Shimmer Overview
+
+The [WinUI Shimmer control](https://www.syncfusion.com/winui-controls/SfShimmer) enhances the responsiveness of your application by displaying a shimmer effect during content loading or background processing.
+
+![WinUI Shimmer control overview](SfShimmer_images/Overview_images/winui_shimmer_overview.gif)
+
+## Key Features
+
+* `Built-in Views`: Choose from various built-in shimmer views, including Circle Persona, Square Persona, Profile, Article, Video, Feed and Shopping.
+* `Wave Directions`: Define animation wave directions, including Default, Left-to-right, Right-to-left, Top-to-bottom and Bottom-to-top.
```

Comment:
@GilbertmanojSF3971 Remove the Wave directions related contents and images.

Reply:
@Ganesh-Mariappan  Removed the WaveDirections related content.